client/js/components/Calculator/CalculatorContainer.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+import Calculator from "./Calculator";
+class CalculatorContainer extends Component {
+  static navigationOptions = {
+    title: "Saving"
+  };
+  constructor(props) {
+    super(props);
+  }
+  render() {
```

Comment:
Component dir should not have a container file. It should just have presentation files with presentation components. Please make the changes.